src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ foreach(schema IN LISTS Schemas)
       COMMAND ${SRC_FOLDER}/external/flatbuffers/build/flatc
       ARGS --cpp
       ARGS -o "${CMAKE_CURRENT_SOURCE_DIR}/include" "${FILE}"
+      ARGS --no-warnings
```

Comment:
Shouldn't we also set the flag in the `Makefile` for C flatbuffers and `src/wazuh_modules/syscollector/CMakeLists.txt` for syscollector flatbuffer in C++?

Reply:
done

Reply:
@matias-braida What about flatcc for C, it also compiles syscollector flatbuffers in Makefile?

Reply:
Flatbuffers in C whose compiler is flatcc executable, don't have any option to suppress warnings.
But on the other hand, on the flatcc headers compilation, there aren't any warnings.
![image](https://github.com/wazuh/wazuh/assets/101227434/9fbf5d64-6e87-4020-9404-367a9d8bf85d)